README.md

```diff
@@ -61,4 +61,209 @@ The third option is relevant when you are planning to implement read-only
 inspection features.
 
 ## API
-Check `AmoClient` in `index.d.ts`
+
+Requests can be made by passing the releavant paramaters to `AmoClient`.
+
+You can check types in `index.d.ts`
+
+## Types
```

Comment:
We need to be more "kind". Add a reference to rpc.md document and index.d.ts here. Add a guide for the developers who do not use Typescript.